org.openscopeproject.TrguiNG.yml

```diff
@@ -0,0 +1,121 @@
+id: org.openscopeproject.TrguiNG
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: trgui-ng
+rename-icon: trgui-ng
+copy-icon: true
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
```

Comment:
do you really need these dbus names?

Reply:
The app has desktop notifications functionality.

Reply:
that doesn't answer the question.

these days toolkit properly use the portal.

so are they really needed?

Reply:
Let me test without them to double check.

Reply:
Testing shows that org.freedesktop.Notifications is needed, otherwise desktop notifications break.
But org.kde.StatusNotifierWatcher is less clear cut:
* Without it tray icon works on x11 and not on wayland
* With it tray icon is semi-broken on KDE on both x11 and wayland: it exists in the tray but has no icon, but works fine on Cinnamon (x11).

Since it's less broken without this permission I have removed it.